docs/howtos/quick-start/cli/_cli-secondary-indexing.mdx

```diff
@@ -0,0 +1,152 @@
+
+Redis Stack enables the JSON data type in Redis.
+
+```shell
+# syntax : set an object value to a key
+> JSON.SET objKey $ value
+
+# example
+> JSON.SET person $ '{"name":"Leonard Cohen","dob":1478476800,"isActive": true, "hobbies":["music", "cricket"]}'
+
+#------------
+# syntax : get object value of a key
+> JSON.GET objKey $
+
+# example
+> JSON.GET person $
+
+#------------
+# syntax : find object key length
+> JSON.OBJLEN objKey $
+
+# example
+> JSON.OBJLEN person $
+
+#------------
+# syntax : find object keys
+> JSON.OBJKEYS objKey $
+
+# example
+> JSON.OBJKEYS person $
+
+#------------
+# syntax : update nested property
+> JSON.SET objKey $.prop value
+
+# example
+> JSON.SET person $.name '"Alex"'
+
+#------------
+# syntax : update nested array
+> JSON.SET objKey $.arrayProp fullValue
+> JSON.SET objKey $.arrayProp[index] value
+
+# example
+> JSON.SET person $.hobbies '["music", "cricket"]'
+> JSON.SET person $.hobbies[1] '"dance"'
+
+#------------
+# syntax : remove nested array item by index
+> JSON.ARRPOP objKey $.arrayProp index
+
+# example
+> JSON.ARRPOP person $.hobbies 1
+```
+
+More details can be found in the [Redis Stack docs](https://redis.io/docs/stack/json/)
+
+---
+
+Redis Stack enables a query and indexing engine for Redis, providing secondary indexing, full-text search and aggregations capabilities.
+
+- We have to create index on schema to be able to search on its data
+
+```shell
+# syntax
+> FT.CREATE {index_name} ON JSON SCHEMA {json_path} AS {attribute} {type}
+# note : attribute = logical name,  json_path = JSONPath expressions
+
+# example
+> FT.CREATE userIdx ON JSON SCHEMA $.user.name AS name TEXT $.user.hobbies AS hobbies TAG $.user.age as age NUMERIC
```

Comment:
Might be helpful to new users to see the `PREFIX` option here as it is used a lot in our documentation and other examples.